agent/f5/oslbaasv1agent/drivers/bigip/lbaas_bigiq.py

```diff
@@ -21,7 +21,7 @@
     from oslo_log import log as logging
 from neutron.common.exceptions import InvalidConfigurationOption
 from neutron.plugins.common import constants as plugin_const
-import novaclient.v1_1.client as nvclient
+import novaclient.v2.client as nvclient
```

Comment:
We only know that Red Hat has slid OSPv8 forward to a new novaclient.  Though less than desirable, this change should be a try/except on v2 and fall back to v1_1 if not present.  We must preserve compatibility with other distros that honor and lock the major version for upgrades within an OpenStack release.